### DISCUSSION TITLE
ModuleNotFoundError: No module named 'pkg_resources'

### DISCUSSION THREAD

**Discussion by JGStuff** [KernelWanderers/OCSysInfo#36] [General]
Hi,
I have been beating my head agains the monitor for days on this.  I followed all directions to download the dependancies and continue to get this error:

jeff@bobs-iMac OCSysinfo % python3.12 main.py                           
Traceback (most recent call last):
  File "/Users/Jeff/OCSysInfo/main.py", line 5, in <module>
    main()
  File "/Users/Jeff/OCSysInfo/src/main.py", line 4, in main
    from src.util.missing_dep import Requirements
  File "/Users/Jeff/OCSysInfo/src/util/missing_dep.py", line 3, in <module>
    from pkg_resources import require, DistributionNotFound
ModuleNotFoundError: No module named 'pkg_resources'

Any suggestions would be appreciated.

Jeff

**Comment by kernel-dev** [comment]
Hi, first and foremost: thank you for bringing this up. Secondly, this should be a part of the [issues](https://github.com/KernelWanderers/OCSysInfo/issues) tab, but don’t worry about that now.

This seems to be related to Python 3.12 not having `setuptools` preinstalled. I’ll update the `requirements.txt` file in accordance to this, and it should be working from hereon out. Thanks for bringing this to our attention once again!

**Comment by JGStuff** [comment]
Thanks for responding to me directly.  Unfortunately, I will be traveling
for the next 2 weeks and will not get to try it out until then.

Jeff
